ui/apps/pixano/src/routes/+page.ts

```diff
@@ -1,3 +1,4 @@
 // since there's no dynamic data here, we can prerender
```

Comment:
remove comment since prerender = false ?

Reply:
Noted, will remove that comment in another branch

Reply:
Fixed in pixano/pixano@9b8e4ee0d9fecc35037a8e113a15c968c7ef6b99 (#29)